wiki/Ranking_Criteria/osu!/en.md

```diff
@@ -1,45 +1,206 @@
+-  **Avoid spinners less than 4 beats.** Players need time to recognize that they have to begin spinning.
+
+#### Difficulty Setting Guidelines
+
+-  Approach Rate should be 5 or less.
+-  Overall Difficulty / HP Drain Rate should be between 1 and 3.
+
+### ![Normal](/wiki/shared/Normal-s.png "Normal") Normal
+
+#### Rules
+
+-  **If stacks are used, Stack Leniency must be set high enough for 1/1 hitobjects to stack.** Directly overlapping hitobjects cause reading problems for new players.
+
+#### Guidelines
+
+-  **Time-distance equality should be used.** Variations are acceptable if they are clearly different from spacing used for different rhythms. When hitobjects are more than 1 beat apart and have relatively high spacing on the playfield, time-distance equality does not need to be as precise.
+-  **When distance snap is used, try to keep it between 0.8x and 1.3x.** Values too much higher or lower may make the difference between slider velocity and spacing counterintuitive. A higher distance snap can be used to avoid overlapping, but the slider velocity should be increased if you find yourself doing this often.
+-  **Note density should consist of mostly 1/1, occasional 1/2, or slower rhythms.**
+-  **Avoid overlapping circles, slider heads, and slider tails.** These can be confusing for new players. 1/2 overlaps due to distance snapping are an exception.
+-  **Avoid long chains of hitobjects with 1/2 gaps.** Too many consecutive hitobjects overlapping can be confusing for new players.
+-  **Avoid 1/2 hitobjects after sliders which appear underneath the previous slider's path.** Players at this level fully follow sliders’ paths, so placing a hitobject directly after the slider is more intuitive. This also includes stacks underneath slider ends.
+-  **Avoid 1/2 sliders with multiple reverses.** New players are unable to read additional reverses because they are visible for such a short amount of time.
+-  **Avoid slider-only sections.** Aiming and following a lot of sliders in a row can be tiring for new players. In such cases, circles and rest moments without hitobjects to click or follow should be used.
+-  **Stacks are acceptable, but avoid switching between 1/1 and 1/2 stacks repeatedly.** This may cause reading issues for new players, since two different rhythms are visually the same.
+-  **Manipulating slider velocity is discouraged.** Slider velocity should only be changed for sections of music with different pacings and should not be drastically variable.
+-  **Use straightforward and easily understandable slider shapes.** New players may not understand how to follow more complex shapes.
+-  **Spinner recovery time should be at least 2 beats.** This is to ensure adequate time to click a hitobject following a spinner.
+-  **Avoid spinners less than 3 beats.** Players need time to recognize that they have to begin spinning.
```

Comment:
They're questioning the fact that more experienced players supposedly don't need as long to start spinning.